data/json/furniture.json

```diff
@@ -949,7 +950,8 @@
             "items": [
                 { "item": "scrap", "count": [2, 8] },
                 { "item": "steel_chunk", "count": [0, 3] },
-                { "item": "pipe", "count": 1 }
+                { "item": "pipe", "count": 1 },
+                { "item": "sheet_metal", "count": [0, 1] }
```

Comment:
It's excessive for deconstruction, because you naturally destruct the structure.

Reply:
Done.